challenge1.exs

```diff
@@ -1,3 +1,65 @@
 defmodule Challenge1 do
+  def divisible3(a, b) do
+    inicio = inicio(a, 0, "")
+    secuencia = secuencia(a, b, inicio, [inicio])
+    divisible(secuencia, [], 0)
+  end
 
+  # Obtener el primer datos de la secuencia
+  def inicio(a, aux, str) when aux == a do
+    String.to_integer(str)
+  end
+
+  def inicio(a, aux, str) when aux < a do
+    aux = aux + 1
+    str = str <> "#{aux}"
+    inicio(a, aux, str)
```

Comment:
It's easier to do
```suggestion
    inicio(a, aux, "#{str}#{aux}")
```

or even 

```suggestion
    inicio(a, aux + 1, "#{str}#{aux + 1}")
```
without having the aux overwriting, what do you think about it?